src/huggingface_hub/hub_mixin.py

```diff
@@ -61,47 +185,31 @@ def save_pretrained(
                 specify config (must be dict) incase you want to save it.
             push_to_hub (:obj:`bool`, `optional`, defaults to :obj:`False`):
                 Set it to `True` in case you want to push your weights to huggingface_hub
-            model_id (:obj:`str`, `optional`, defaults to :obj:`save_directory`):
```

Comment:
also removed this docstring here. kwargs encapsulates this, eh? I'd understand why you might want this here for clarity, though.